app/controllers/application_controller.rb

```diff
@@ -80,6 +80,14 @@ def sublayout
     end
   end
 
+  def set_cache_headers
```

Comment:
Maybe `set_no_cache_headers` ? 
I don't know what's better, but probably it better reflects the meaning :grimacing: